packages/core/src/renderer/components/status-bar/status-bar.module.scss

```diff
@@ -13,7 +13,7 @@
 
   display: inline-grid;
   grid-template-columns: 1fr 1fr;
-  background-color: var(--colorInfo);
+  background-color: var(--primary);
```

Comment:
What's intended `--colorInfo` variable?

Reply:
All colors needs to be reviewed and cleaned up.